modules/programs/watson.nix

```
@@ -0,0 +1,110 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.programs.watson;
+
+  iniFormat = pkgs.formats.ini { };
+
+  configDir = if pkgs.stdenv.isDarwin then
+    "Library/Application Support"
+  else
+    config.xdg.configHome;
+
+in {
+  options = {
+    programs.watson = {
+      enable = mkEnableOption ''
+        watson, a wonderful CLI to track your time!
+      '';
+
+      package = mkOption {
+        type = types.package;
+        default = pkgs.watson;
+        defaultText = literalExpression "pkgs.watson";
+        description = "Package providing <command>watson</command>.";
+      };
+
+      enableBashIntegration = mkOption {
+        type = types.bool;
+        default = true;
+        description = ''
+          Whether to enable Watson's Bash integration.
+        '';
+      };
+
+      enableZshIntegration = mkOption {
+        type = types.bool;
+        default = true;
+        description = ''
+          Whether to enable Watson's Zsh integration.
+        '';
+      };
+
+      enableFishIntegration = mkOption {
+        type = types.bool;
+        default = true;
+        description = ''
+          Whether to enable Watson's Fish integration.
+        '';
+      };
```

these should use `mkEnableOption`:

```suggestion
      enableBashIntegration = mkEnableOption "watson's bash integration" // {
        default = true;
      };

      enableZshIntegration = mkEnableOption "watson's zsh integration" // {
        default = true;
      };

      enableFishIntegration = mkEnableOption "watson's fish integration" // {
        default = true;
      };
```